pkg/machinery/config/config_schema_test.go

```diff
@@ -2,11 +2,11 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-package v1alpha1_test
+package config_test
 
-/* Commented out to workaround an issue with go1.20.5.
```

Comment:
This file was totally commented out - brought it back and added tests for other document types (only `NetworkRuleConfig` for now)